src/Umbraco.Cms.Api.Management/Controllers/Document/DocumentControllerBase.cs

```diff
@@ -12,4 +12,10 @@ namespace Umbraco.Cms.Api.Management.Controllers.Document;
 public abstract class DocumentControllerBase : ContentControllerBase
 {
     protected IActionResult DocumentNotFound() => NotFound("The requested Document could not be found");
+
+    protected IActionResult ContentTypeOperationStatusResult(ContentTypeOperationStatus contentTypeOperationStatus) =>
+        contentTypeOperationStatus switch
+        {
+            ContentTypeOperationStatus.NotFound => NotFound("The document type with the given key was not found"),
+        };
```

Comment:
This is a little wierd, as we have to handle DocumentTypes in the document controller, not sure if this would just make sense in the controller instead

Reply:
True, but this is a symptom on the error about the ID/key is actually a document Id.